plugins/modules/cloud/misc/proxmox_instance_actions.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2020, Tristan Le Guern <tleguern at bouledef.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: proxmox_instance_actions
+short_description: Perform actions on Qemu/KVM and LXC instances from Proxmox VE
+version_added: 2.0.0 ?
+description:
+  - Perform actions on an existing instance from Proxmox VE.
+  - This module does not return data.
+options:
+  action:
+    description:
+      - Perform the given action.
+      - C(reset) is only valid with Qemu/KVM instances.
+    type: str
+    choices: ['reboot', 'reset', 'resume', 'shutdown', 'start', 'stop', 'suspend']
+    required: true
+  force:
+    description:
+      - Force an instance to shutdown.
+      - Only possible if I(action) is C(shutdown).
```

Comment:
```suggestion
      - Only possible if I(action) is C(shutdown).
```
shouldn't we check this in the code or we could add something like
```
 - Only possible if I(action) is C(shutdown), otherwise, ignored.
```

Reply:
I will add “otherwise, ignored”, it's a good idea.